### PR TITLE
57754 cleanup and update tests

### PR DESCRIPTION
**Pull request by fazilqa**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

- This PR cleanups unused pages and adds more tests for Patient Interstitial page to verify links of Crisis Line modal window.

## Related issue(s)

- https://jira.devops.va.gov/browse/MHV-57754

## Testing done

- Relevant tests have been executed in headed mode with a 100x pattern.
- All tests have been successfully conducted in headless mode without any failures.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import mockToggles from '../fixtures/toggles-response.json';
 import { Locators } from '../utils/constants';
 
 class GeneralFunctionsPage {
+  get header() {
+    return cy.get(`h1`);
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js:6]: I think this linter warning here is for a reason. Please see if we need to take another look

File: src/applications/mhv-secure-messaging/tests/e2e/pages/PatientInterstitialPage.js [hunk 2]
```diff
@@ -1,6 +1,18 @@
 import { Locators } from '../utils/constants';
 
 class PatientInterstitialPage {
+  get crisisLineLink() {
+    return cy.get(`[text="Connect with the Veterans Crisis Line"]`);
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/tests/e2e/pages/PatientInterstitialPage.js:5]: same as above

**Review by alex-morgun-1 — CHANGES_REQUESTED** [review]
@fazilqa there are few linter warning that I believe we are expected to address. Would you mind taking another look please?

File: src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js (hunk 1 above)
Comment (fazilqa) [reply src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js:6]: fixed

Comment (oleksii-morgun) [reply src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js:6]: I am still  seeing linter warning. I think it implies to use `this` or you may transform this to an arrow function

Comment (fazilqa) [reply src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js:6]: transformed to arrow functions

**Review by alex-morgun-1 — APPROVED** [review]
looks good

**Pull request merged**